tests/wpt/mozilla/meta/MANIFEST.json

```diff
@@ -776,6 +776,18 @@
             "url": "/_mozilla/css/border_radius_clip_a.html"
           }
         ],
+        "css/border_radius_clipping_contents_a.html": [
+          {
+            "path": "css/border_radius_clipping_contents_a.html",
+            "references": [
+              [
+                "/_mozilla/css/border_radius_clipping_contents_ref.html",
+                "=="
+              ]
+            ],
+            "url": "/_mozilla/css/border_radius_clipping_contents_a.html"
+          }
+        ],
```

Comment:
This is missing a second entry later on in the file. It's much easier to use `./mach create-wpt --reftest path/to/test.html --reference path/to/reference.html` which does everything right the first time.